### PR TITLE
feat: categorize assets by blockchain refactor

### PR DESCRIPTION
**Pull request by EbrahimUmee**

### CODE REVIEW

**Comment by UmeeAndrey** [comment]
@EbrahimUmee  @brandoncomer 

In order to support the chain functionality and keep the repository and data organized can we please do the following:    

- each chain is represented by a directory. 
- each directory contains **info.json**  and a **token-list.json** files. 

We should enforce the following interfaces:
1 - Token Interface: 
      ` name
        denom
        baseDenom
        decimals
        symbol
        address
        logoURI
      `
2 - Chain Info:
      ` name
       chain_id
       decimals
       symbol
       staking_tokens:  Token [ ]
       type: COSMOS | ETHEREUM | DAI ect ... 
       website
       description
       explorer
     `
 3 - Token List Json:
       
     `{
        tokens: Token[ ]
     }`

File: blockchains/osmo.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Juno",
```
Comment (humarkx) [inline blockchains/osmo.json:2]: This should be:
```suggestion
  "name": "Osmo",
```

Comment (UmeeAndrey) [inline blockchains/osmo.json]:
```suggestion
  "name": "Osmo",
```

**Comment by UmeeAndrey** [comment]
@EbrahimUmee  can you please work on refactoring the tests for this ?

**Review by UmeeAndrey — APPROVED** [review]

**Pull request merged**